docs/2.8.5/docs/canton/usermanual/pruning.rst

```diff
@@ -73,8 +73,10 @@ Best Practices
 - A catastrophic failure of a participant and its backup can be mitigated by rebuilding its state from the sequencer by
   replaying messages. However, this becomes impossible once the required messages have been pruned from the sequencer.
   For this reason, the backup strategy for participant nodes should be coordinated with the sequencer’s pruning schedule.
-- For high availability nodes that share a common database, the pruning schedule has to be set on an active replica
-  (participant, mediator) or one active shard (database sequencer).
+- For high-availability nodes that share a common database, the pruning schedule has to be set on an active participant
```

Comment:
```suggestion
- For high-availability nodes that share a common database, the pruning schedule has to be set on an active participant,
```

Reply:
As discussed, we modified the sentence to:

"For high-availability nodes that share a common database, the pruning schedule has to be set on the active participant
  replica, the active mediator replica, or the admin sequencer."